### PR TITLE
updated readme and parameter descriptions

### PR DESCRIPTION
**Pull request by BenGriffith**

### CODE REVIEW

File: tollway/constants.py [hunk 1]
```diff
@@ -8,3 +10,13 @@
 INCLUDE_LATE_RATE = 20
 INCLUDE_DUPLICATE_RATE = 50
 ALL_EVENTS_COUNT = 250
+
+
+class Help(Enum):
```
Comment (bbelderbos) [inline tollway/constants.py:15]: uppercase for constant?

File: tollway/__main__.py [hunk 2]
```diff
@@ -1,31 +1,41 @@
...
-        total_events: Optional[int] = typer.Option(default=1),
-        event_rate: Optional[float] = typer.Option(default=1.0),
-        output_file: Optional[bool] = typer.Option(default=False),
-        output_filename: Optional[str] = typer.Option(default="tollway-traffic.json"),
-        date_variation: Optional[bool] = typer.Option(default=False),
-        include_late: Optional[bool] = typer.Option(default=False),
-        include_duplicate: Optional[bool] = typer.Option(default=False),
-        ):
+    total_events: Annotated[int, typer.Option(help=Help.TOTAL_EVENTS.value)] = 1,
+    event_rate: Annotated[float, typer.Option(help=Help.EVENT_RATE.value)] = 1.0,
```
Comment (bbelderbos) [inline tollway/__main__.py:26]: have not used Help enum, nice thing is that you have all the strings in one place now!

File: tollway/constants.py (hunk 1 above)
Comment (BenGriffith) [reply tollway/constants.py:15]: yes, was thinking do uppercase since being used as a constant

File: tollway/__main__.py (hunk 2 above)
Comment (BenGriffith) [reply tollway/__main__.py:26]: while they're somewhat related, main thought was it would be nice to have all strings in one place

**Pull request merged**